linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/tests/incorrect_max_pass_age.fail.sh

```diff
@@ -1,13 +1,7 @@
 #!/bin/bash
 
```

Comment:
nitpick: Could you remove this extra empty line, please?

Reply:
> nitpick: Could you remove this extra empty line, please?

all the others scripts usually have a empty line between the shebang and variables, is this really needed?